tests/Functional/ProductManagerTest.php

```diff
@@ -319,18 +327,15 @@ public function testItReturnsACollectionOfProductsFromParameters(): void
         $this->assertContainsOnlyInstancesOf(Product::class, $result);
     }
 
-    public function testItReturnsAFeedResponseFromAProductCreateRequest(): void
+    /**
+     * @dataProvider productActions
+     */
+    public function testItReturnsAFeedResponseFromAProductActionRequest($action, $isGlobal): void
```

Comment:
It miss define type of variable it will receive as parameter

Reply:
Fixed